content/standards/UPRN/index.html.md

```diff
@@ -1,16 +1,20 @@
 ---
-title: UPRN
 name: UPRN
-organisation: Ordnance Survey
+contraction: UPRN
+specification:
+  name: 'Identifying property and street information'
+  url: https://www.gov.uk/government/publications/open-standards-for-government/identifying-property-and-street-information
```

Comment:
This is not the specification for UPRNs (nor USRNs). Similar to other comments on accuracy of the data, ignore it for this PR if the plan is to address these in another PR.

Reply:
Very fair, this is existing, but I'll amend to the actual spec